ProjectWithUMLDiagram/src/com/beyond/person/core/helper/Help.java

```diff
@@ -0,0 +1,25 @@
+package com.beyond.person.core.helper;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Help {
+    public static boolean checkAlphabet(String name) {
```

Comment:
@AraChilingaryan you can use match and regex 
public boolean isAlpha(String name) {
    return name.matches("[a-zA-Z]+");
}

Reply:
done